shopinvader/components/product_product_event_listener.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.component.core import Component
+from odoo.addons.component_event import skip_if
+
+
+class ProductProductEventListener(Component):
+    _name = 'product.product.event.listener'
+    _inherit = 'base.connector.listener'
```

Comment:
@acsonefho You should have added the dependency on 'connector' module as this is defined there. Tests are failing when testing only 'shopinvader' module. Can you fix it ?